lfroms/COMP4109-Project
# Create basic chat ui

### What is the problem being solved in this PR?



Creating basic UI for chat page

### Related issues or PRs



Resolves #11

### How did you accomplish this and why did you choose this approach?



Created new file [chatID] within a chat folder, to handle the query param and get the specific chat room page. The messages added are saved temporarily with a hook array. Messages are created by using a map on the conversation text.

### How to test changes



Navigate to page localhost:3000/chat/:id and you should see the page appear, displaying with chatroom it is. When you type a message and hit send, it will add it to the list of messages. When the page is reloaded, the default messages will reappear.

### Checklist
